libs/angular/tsconfig.lib.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "rootDir": ".",
```

Comment:
I think this can be removed, if it is correctly inferred from the "include" glob?

```suggestion
```

Reply:
Yes, good catch 👌🏻